android-project/app/src/main/java/org/libsdl/app/SDLActivity.java

```diff
@@ -1938,6 +1939,18 @@ public void run() {
         }
         return 0;
     }
+
+    /**
+     * This method is called by SDL using JNI.
+     */
+    public static int openFileDescriptor(String uri, String mode) throws Exception {
```

Comment:
I think this function needs to be added to the proguard file

Reply:
Good catch. Added.